sdks/python/apache_beam/runners/dataflow/internal/apiclient.py

```diff
@@ -314,8 +314,15 @@ def __init__(
         if container_image_url in already_added_containers:
           # Do not add the pipeline environment again.
 
+          # Currently, Dataflow uses Docker container images to uniquely
+          # identify execution environments. Hence Dataflow executes all
+          # transforms that specify the same Docker container image in the
+          # same container. Dependencies of all environments that specify a
```

Comment:
`the same Docker container image in the same container` sounds little confusing to me. How about something like `the same Docker container image in the single container instance`?